changelogs/unreleased/protobuf_encoder_module.md

```diff
@@ -0,0 +1,4 @@
+## feature/lua
+
+* Introduced Lua implementation of protobuf encoder.
+  Four main protobuf types are supported.
```

Comment:
From the commit message:

> Protobuf decoder API

It looks like a typo, because an encoder is implemented here.

----

While we're here, I've another question. In the commit message, pull request description and the changelog entry you're speaking about 'four main protobuf types', but it doesn't tell us much without a proper context.

We can say 'all the Protocol Buffers wire types are supported except group start/end, which are deprecated in proto3'. It brings enough context for a reader.

OTOH, is it important enough to highlight it everywhere? I would just say that an encoder is introduced. No need to dive a reader into the wire types of the protocol, because they all are supported (or deprecated).

We can strip it from the changelog entry and leave in the commit message/pull request description as a separate note.

> Note: Protocol Buffers defines six wire types \[1]. VARINT, I64, LEN, I32 are supported by the encoder. Deprecated SGROUP and EGROUP are not implemented and there are no plans to implement them.
>
> \[1]: https://protobuf.dev/programming-guides/encoding/#structure

Reply:
Fixed

Reply:
I deleted mention of four types

Reply:
AFAIS, not all the mentions of 'four type' and/or 'main types' are removed. Please, double check the commit message text and the pull request description.

Reply:
Seems solved. Please, respond to comments, when they're fixed.